### PR TITLE
LOK-2477 : New syslog config task set and publish to minion.

### PR DESCRIPTION
**Pull request by MusaidAli**
## Description
New syslog config task set and publish to minion

## Jira link(s)
-https://opennms.atlassian.net/browse/LOK-2477

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: inventory/main/src/main/java/org/opennms/horizon/inventory/service/syslogconfig/SyslogConfigBean.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ */
+package org.opennms.horizon.inventory.service.syslogconfig;
+
+public class SyslogConfigBean {
+
+    private String syslogAddress;
+    private Integer syslogPort;
```
Comment (fooker) [inline inventory/main/src/main/java/org/opennms/horizon/inventory/service/syslogconfig/SyslogConfigBean.java:27]: Consider renaming to `listenAddress` and `listenPort` for more concise naming.

**Review by fooker — CHANGES_REQUESTED** [review]

File: inventory/main/src/main/java/org/opennms/horizon/inventory/service/syslogconfig/SyslogConfigBean.java (hunk 1 above)
Comment (MusaidAli) [reply inventory/main/src/main/java/org/opennms/horizon/inventory/service/syslogconfig/SyslogConfigBean.java]: renamed these parameters .

**Review by cgorantla — CHANGES_REQUESTED** [review]
This config task needs to be published on Minion and Syslog listener should use syslog port passed from here.

**Comment by smunir-onms** [comment]
replaced with https://github.com/OpenNMS-Cloud/lokahi/pull/1888

**Pull request closed without merging**